modules/geo-layers/src/tile-layer/tile-layer.js

```diff
@@ -1,23 +1,21 @@
 import {CompositeLayer} from '@deck.gl/core';
 import {GeoJsonLayer} from '@deck.gl/layers';
 
-import Tileset2D, {STRATEGY_DEFAULT} from './utils/tileset-2d';
-import {tileToBoundingBox as defaultTile2Bbox} from './utils/tile-util';
-import {getTileIndices as defaultGetIndices} from './utils/viewport-util';
+import Tileset2D, {STRATEGY_DEFAULT} from './tileset-2d';
 
 const defaultProps = {
   renderSubLayers: {type: 'function', value: props => new GeoJsonLayer(props), compare: false},
   getTileData: {type: 'function', value: ({x, y, z}) => null, compare: false},
-  tileToBoundingBox: {type: 'function', value: defaultTile2Bbox, compare: false},
-  getTileIndices: {type: 'function', value: defaultGetIndices, compare: false},
+  Tileset: Tileset2D,
```

Comment:
Nit: Don't we want to we accept a tileset instance, rather than a constructor? That seems a bit limiting?

Of course if none is provided we instantiate Tileset2D

Reply:
Removing for now, will re-visit when Tileset2D is moved to loaders.gl.